assets/src/edit-story/migration/migrations/v0020_isFillDeprecate.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function isFillDeprecate({ pages, ...rest }) {
+  return {
+    pages: pages.map(reducePage),
+    ...rest,
+  };
+}
+
+function reducePage({ elements, ...rest }) {
+  return {
+    elements: elements.map(updateElement),
+    ...rest,
+  };
+}
+
+function updateElement(element) {
+  delete element.isFill;
```

Comment:
So if my story had a fill element, it is now converted to a regular element at whatever size it had before being set to fill? Shouldn't we at least full-size the element so it appears more or less as before?

Reply:
TBH I purposefully didn't care about this since it didn't seem this featured was really used too much and we haven't released yet. I can definitely add this though for the purpose of good practice, will do!